plans/feat-accounting-manage-accounts.md

```
@@ -0,0 +1,248 @@
+
+```ts
+{ close: []; changed: [] }
+```
+
+`changed` fires after every successful `upsertAccount`. Parent forms re-emit upward so `View.vue` can refetch (covered by Section 3 — but the explicit emit is kept as a fallback in case pub/sub is unavailable in some test contexts).
+
+**Layout**
+
+Modal uses the same overlay pattern as `NewBookForm.vue` (modal mode) — `fixed inset-0 z-50 bg-black/20 flex items-center justify-center`. Backdrop click + `Esc` key + Cancel button all close. Body is a fixed-width card (`w-[28rem]` or so), max-height capped to viewport with scrolling so a long chart of accounts doesn't blow the layout.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)